index.html

```diff
@@ -1,162 +1,10 @@
-<p>If you don't like these rules, raw patches are more than welcome!</p>
-
-
-<h4>Bugs</h4>
-
-<p>Please be sure to report bugs <a class="reference external" target="_blank" href="http://www.chiark.greenend.org.uk/~sgtatham/bugs.html">as effectively as possible</a>, to ensure that we understand and act on them quickly.</p>
-
-<p>Usually <a class="reference external" target="_blank" href="https://github.com/eventlet/eventlet/issues/new">open issue at Github</a></p>
-
-<p>In special cases, to contact current maintainers directly, look up info on Github project page.</p>
-
-
-<div class="section" id="web-crawler-example">
-<h2>Web Crawler Example<a class="headerlink" href="#web-crawler-example" title="Permalink to this headline">¶</a></h2>
-<p>This is a simple web &#8220;crawler&#8221; that fetches a bunch of urls using a coroutine pool.  It has as much concurrency (i.e. pages being fetched simultaneously) as coroutines in the pool.</p>
-
-<pre><code class="language-python">import eventlet
-# note: this urllib import doesn't work in Python2
-from eventlet.green.urllib.request import urlopen
-
-
-urls = [
-    "http://www.google.com/intl/en_ALL/images/logo.gif",
-    "https://wiki.secondlife.com/w/images/secondlife.jpg",
-    "http://us.i1.yimg.com/us.yimg.com/i/ww/beta/y3.gif",
-]
-
-
-def fetch(url):
-    return urlopen(url).read()
-
-
-pool = eventlet.GreenPool()
-
-for body in pool.imap(fetch, urls):
-    print("got body", len(body))
-</code></pre>
-
-
-<h3>Flair</h3>
-<p>
-  <a class="reference external" target="_blank" href="https://pypi.org/project/eventlet/" rel="nofollow"><img alt="PyPI" src="https://img.shields.io/pypi/v/eventlet"></a>
-  <a class="reference external" target="_blank" href="https://travis-ci.org/eventlet/eventlet" rel="nofollow"><img alt="Travis build" src="https://travis-ci.org/eventlet/eventlet.svg?branch=master"></a>
-  <a class="reference external" target="_blank" href="https://codecov.io/gh/eventlet/eventlet" rel="nofollow"><img alt="Codecov coverage" src="https://codecov.io/gh/eventlet/eventlet/branch/master/graph/badge.svg?token=k01TsER6fy"></a>
-</p>
-</div>
-</div>
-</div>
-
-
-<div class="section" id="contents">
-</div>
-</div>
-</div>
-</div>
-<div class="sphinxsidebar">
-<div class="sphinxsidebarwrapper">
-<h3>Links</h3>
-<ul>
-<li><a class="reference external" href="doc/">Documentation</a></li>
-<li><a class="reference external" href="doc/changelog.html">Changelog</a></li>
-<li><a class="reference external" target="_blank" href="https://github.com/eventlet/eventlet/">Eventlet on Github</a></li>
-<li><a class="reference external" target="_blank" href="irc://chat.freenode.net/#eventlet">IRC channel</a></li>
-<li><a class="reference external" target="_blank" href="https://www.openhub.net/p/eventlet">Open Hub stats</a></li>
-</ul>
-        </div>
-      </div>
-    </div>
-    <script src="//yandex.st/highlightjs/7.3/highlight.min.js"></script>
-    <script>hljs.initHighlightingOnLoad();</script>
-  </body>
+<html>
```

Comment:
Maybe also some text with a link, for people with JavaScript disabled?

Reply:
Although I guess this is temporary until DNS gets updated so not a permanent solution anyway.

Reply:
Will submit a follow up patch.